packages/progress-indicator/src/dots/element.css.ts

```diff
@@ -0,0 +1,59 @@
+import { css, unsafeCSS } from 'lit';
+import { styles as theme } from '@equinor/fusion-web-theme';
+
+export const style = css`
+  :host {
+    display: flex;
+    justify-content: center;
+  }
+  :host([color='primary']) {
+    --fwc-dots-progress-color: ${unsafeCSS(theme.colors.infographic.primary__moss_green_100.getVariable('color'))};
+  }
+  :host([color='tertiary']) {
+    --fwc-dots-progress-color: ${unsafeCSS(theme.colors.interactive.danger__resting.getVariable('color'))};
+  }
+  :host([color='neutral']) {
+    --fwc-dots-progress-color: ${unsafeCSS(theme.colors.interactive.icon_on_interactive_colors.getVariable('color'))};
+  }
+`;
+
+const sizeStyle = css`
+  :host([size='small']) {
+    --fwc-circular-progress-size: 32px;
+  }
+  :host([size='medium']) {
+    --fwc-circular-progress-size: 48px;
+  }
+  :host([size='large']) {
+    --fwc-circular-progress-size: 64px;
+  }
+`;
+
+const graphicsStyles = css`
+  svg {
+    fill: var(--fwc-dots-progress-color, currentColor);
+    height: var(calc(--fwc-circular-progress-size / 4), calc(1.2em / 4));
```

Comment:
Wops, I have used --fwc-**circular**-progress-size as variable name. Will change to --fwc-dots-progress-size in a later commit, after reviews to potentially save chromatic builds..